platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -34,6 +34,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * This factory helps construct clients for secure restful communication.
+ * Since there is no open standard for rest security, clients should only be constructed
+ * with a subject when the message is or may be sent to another DDF. No other system
+ * would know what to do with a base64 encoded saml assertion.
```

Comment:
I'm not sure I completely agree with this comment. There is a standard for REST security between two NPEs and that would be the SAML ECP profile, we just don't implement it yet. Also sending a SAML assertion in the Authorization header is a standard practice, it is just that most systems out there don't do SAML because "it is hard".